src/clusterfuzz/_internal/bot/tasks/utasks/regression_task.py

```diff
@@ -255,21 +255,6 @@ def validate_regression_range(
   return None
 
 
-def handle_low_confidence_in_regression_range(output: uworker_msg_pb2.Output):
```

Comment:
Can you confirm these functions are just moved and not modified?

Reply:
Yes, I confirm! I reordered stuff so all the error handlers would be together.